beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1645,6 +1638,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                 })
         };
 
+        let mut graffiti: Graffiti = Graffiti::default();
+        // Panic-free because graffiti and self.graffiti are both `Graffiti`.
+        graffiti.copy_from_slice(&self.graffiti);
+
```

Comment:
I copied the original code:
https://github.com/sigp/lighthouse/blob/95320f8ab0141a66cd660325a4a4474a82767d70/beacon_node/beacon_chain/src/beacon_chain.rs#L1623-L1624

Wondering if calling `.clone()` would get optimized. This would allow for easier to read code.
One could even argue that EVEN if `.clone()` doesn't get optimized, the change is soooooo small, and done only once at initialization, that I could just use `.clone()`. I hope the reviewer will let me know! :)

Reply:
I ran some benchmarks and it indeed looks like `.clone()` and this are the same. Pushed https://github.com/sigp/lighthouse/pull/1320/commits/699fcc32785e9bafa0b21c26394177307667a7c6 , which makes it easier to read :)